validation/src/main/scala/hmda/validation/rules/lar/macro/Q007.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.`macro`
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.rules.lar.`macro`.MacroEditTypes._
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+object Q007 extends AggregateEditCheck[LoanApplicationRegisterSource, LoanApplicationRegister] {
+
+  val config = ConfigFactory.load()
+  val multiplier = config.getInt("hmda.validation.macro.Q007.numOfLarsMultiplier")
+
+  override def name = "Q007"
+
+  override def apply(lars: LoanApplicationRegisterSource)(implicit system: ActorSystem, materializer: ActorMaterializer, ec: ExecutionContext): Future[Result] = {
+
+    val approvedButNotAccepted =
+      count(lars.filter(lar => lar.actionTakenType == 2))
+
+    val total = count(lars)
```

Comment:
@schbetsy I did think about this, it complicated the implementation on the first pass I did at this and decided to keep it simpler for now. Will try to put something together, see what it looks like. 

@nickgrippin I don't think we can really do that. There is at least one rule that checks that the `totalLines` field in TS is correct (there is no guarantee that this number is correct).